genai-perf/genai_perf/profile_data_parser/llm_profile_data_parser.py

```diff
@@ -161,6 +163,19 @@ def _parse_requests(self, requests: dict) -> Metrics:
             chunked_inter_token_latencies,
         )
 
+        # request goodput
```

Comment:
Could you break the goodput calculation out into its own function that gets called here? This function is quite long and needs to be broken out at some point. I'd rather not extend it further if not needed.